src/ontology/foodon-edit.owl

```diff
@@ -58771,7 +58937,10 @@ SubClassOf(obo:FOODON_03490125 obo:FOODON_03490126)
 # Class: obo:FOODON_03490126 (plastic tray or pan)
 
 AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03490126 obo:IAO_0000428)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03490126 "A flat plastic container with raised edges that is used to carry things."@en)
+AnnotationAssertion(obo:IAO_0000119 obo:FOODON_03490126 "https://www.dictionary.com/browse/tray"@en)
```

Comment:
Lets use Wiktionary instead of dictionary.com if possible.  https://en.wiktionary.org/wiki/tray